app/src/main/java/com/example/vaccinespotter/worker/BackgroundWorker.java

```diff
@@ -0,0 +1,61 @@
+package com.example.vaccinespotter.worker;
+
+import android.content.Context;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.work.Worker;
+import androidx.work.WorkerParameters;
+
+import com.example.vaccinespotter.notifications.VaccineNotificationManager;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Locale;
+
+public class BackgroundWorker extends Worker {
+    private static final String TAG = "BackgroundWorker";
+    private static final int numberOfCycles = 5;

```

Comment:
static members naming convention https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html

If your variable stores a constant value, such as static final int NUM_GEARS = 6, the convention changes slightly, capitalizing every letter and separating subsequent words with the underscore character. By convention, the underscore character is never used elsewhere.